contribs/gmf/src/d3-lidar-profile/lidarprofile.js

```diff
@@ -0,0 +1,37 @@
+goog.provide('gmf.lidarProfile');
+
+/**
+* Provides a D3js component to be used to draw an lidar point cloud
+* profile chart.
+* Requires access to a Pytree webservice: https://github.com/sitn/pytree
+*
+* @constructor
+* @param {Object} options from pytree
```

Comment:
It's possible to provide a better type ?

Reply:
See previous comment ;-)